Directory.Build.props

```diff
@@ -0,0 +1,2 @@
+<Project>
```

Comment:
Please remove this file, as it currently do not serve any purpose.

Reply:
This file is required for this repository to be used as a submodule. It prevents the projects in this repository from inheriting these files from further up in the directory structure.

Reply:
I can remove these files, but it would block certain use cases like this. Let me know.

Reply:
Ah, I see. I was not aware of that.

In that case it is fine to not remove them.